test_ros2cli/test/config_ros2action_test.py

```diff
@@ -29,7 +29,7 @@ def get_action_server_node_action():
     return ExecuteProcess(
```

Comment:
Replace this with:
```python3
Node(
    package='action_tutorials_py',
    node_executable='fibonacci_action_server',
    sigterm_timeout=LaunchConfiguration('sigterm_timeout', default=30)
)
```

Before we were installing an script, and I needed to use `sys.executable`.
Now we are installing an executable.
See https://github.com/ros2/demos/issues/374

Reply:
This should solve the windows problem

Reply:
a36df3a